modules/reference/pages/cluster-properties.adoc

```diff
@@ -1,1796 +1,5372 @@
 
 ---
 
-=== oidc_clock_skew_tolerance
+=== sasl_kerberos_config
 
-The amount of time (in seconds) to allow for when validating the expiry claim in the token.
+The location of the Kerberos `krb5.conf` file for Redpanda.
 
+*Type*: string.
 
-*Units*: seconds
+*Requires Restart:* No
 
-*Default*: 30 sec
+*Nullable:* No
+
+*Visibility:* `user`
+
+*Type:* string
+
+*Default:* `/etc/krb5.conf`
 
 ---
 
-=== oidc_discovery_url
+=== sasl_kerberos_keytab
 
-The URL pointing to the well-known discovery endpoint for the OIDC provider.
+The location of the Kerberos keytab file for Redpanda.
+
+*Type*: string.
 
-*Default*:  https://auth.prd.cloud.redpanda.com/.well-known/openid-configuration
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `user`
+
+*Type:* string
+
+*Default:* `/var/lib/redpanda/redpanda.keytab`
 
 ---
 
-=== oidc_token_audience
+=== sasl_kerberos_principal
 
-A string representing the intended recipient of the token.
+The primary of the Kerberos Service Principal Name (SPN) for Redpanda.
 
-*Type*: string
+*Type*: string.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `user`
 
-*Default*: `redpanda`
+*Type:* string
+
+*Default:* `redpanda`
 
 ---
 
-=== rm_sync_timeout_ms
+=== sasl_kerberos_principal_mapping
 
-Resource manager's synchronization timeout. Maximum time for this node to wait for internal state machine to catch up with all events written by previous leaders before rejecting a request.
+Rules for mapping Kerberos principal names to Redpanda user principals.
 
-*Units*: milliseconds
+*Type*: array of string.
+
+*Requires Restart:* No
+
+*Nullable:* No
 
-*Default*: 10000 (10 sec)
+*Visibility:* `user`
 
-*Restart required*: yes
+*Type:* array
+
+*Default:* `[default]`
 
 ---
 
-=== rpc_server_listen_backlog
+=== sasl_mechanisms
 
-Maximum TCP connection queue length for Kafka server and internal RPC server. If `null` (the default value), no queue length is set.
+A list of supported SASL mechanisms. `SCRAM` and `GSSAPI` are allowed.
+
+*Type*: array of string
 
-*Units*: number of queue entries
 
-*Default*: null
+*Valid values*: `"SCRAM"`, `"GSSAPI"`.
 
-*Range*: [1, ...]
+*Requires Restart:* No
 
-*Restart required*: yes
+*Nullable:* No
+
+*Visibility:* `user`
+
+*Type:* array
+
+*Default:* `[scram]`
 
 ---
 
-=== rpc_server_tcp_recv_buf
+=== segment_appender_flush_timeout_ms
+
+Maximum delay until buffered data is written.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `std::chrono::milliseconds(1s)`
+
+---
+
+=== segment_fallocation_step
+
+Size for segments fallocation.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Default:* `33554432`
+
+---
+
+=== space_management_enable
+
+Option to explicitly disable automatic disk space management. If this was explicitly disabled while using v23.2, this will remain disabled following an upgrade.
```

Comment:
```suggestion
Option to explicitly disable automatic disk space management. If this property was explicitly disabled while using v23.2, it will remain disabled following an upgrade.
```